README.md

```diff
@@ -1,8 +1,13 @@
-# DigitalOcean App Platform Image and DigitalOcean Container Registry publish
-This action can be used to redeploy application on the DigitalOcean's [App Platform](https://www.digitalocean.com/products/app-platform/) using github action. This Action has two use cases one is to redeploy your application on App Platform with same configuration. The other use case is to update the DigitalOcean Container Registry configuration and deploy to App Platform. This github action uses DigitalOcean AppSpec [App Spec](https://docs.digitalocean.com/products/app-platform/references/app-specification-reference/).
+# Deploy a [DigitalOcean App Platform](https://www.digitalocean.com/products/app-platform/) app using GitHub Actions.
```

Comment:
Do we need a disclaimer that there app has to already exist?

Reply:
How about?
# Redeploy a [DigitalOcean App Platform](https://www.digitalocean.com/products/app-platform/) app using GitHub Actions.

Reply:
i think above title can imply that app should be deployed without disclaimer

Reply:
I think it'd be confusing. Let's leave it as is and see if it becomes a problem.